### PR TITLE
Update README.md

### LINKED ISSUE #76: Couldn't run baseEstimatorV1
I'm trying to run the [`baseEstimatorV1`](https://github.com/robotology/whole-body-estimators/tree/38dc6a269f1b0bce09f31d8ae9c4ca3602e8e46b/devices/baseEstimatorV1) device on my setup. 

Following the setup instructions of the README file, I did the following:

1. Start the yarp server
1. Open Gazebo Classic with the clock plugin
1. Insert `iCubGazeboV2_5 (no hands)`
1. Play the simulation
1. Run `YARP_CLOCK=/clock yarprobotinterface --config launch-fbe-analogsens.xml`

<details>
<summary>Click to expand the full output</summary>

```
~ 
❯ YARP_CLOCK=/clock yarprobotinterface --config launch-fbe-analogsens.xml
yarp: Port /IITICUBLAP110/yarprobotinterface/31105/clock:i active at tcp://192.168.8.102:10126/
Success: port-to-port persistent connection added.
yarp: Waiting for clock server to start broadcasting data ...
yarp: Receiving input from /clock to /IITICUBLAP110/yarprobotinterface/31105/clock:i using tcp
||| configuring
||| default config file specified as yarprobotinterface.ini
||| checking [/home/dferigo/yarprobotinterface.ini] (pwd)
||| checking [/iit/sources/robotology-superbuild/build/install/share/yarp/robots/iCubGazeboV2_5/yarprobotinterface.ini] (robot)
||| checking [/iit/sources/robotology-superbuild/build/install/share/iCub/robots/iCubGazeboV2_5/yarprobotinterface.ini] (robot)
||| checking [/iit/sources/robotology-superbuild/build/install/share/ICUBcontrib/robots/iCubGazeboV2_5/yarprobotinterface.ini] (robot)
||| checking [/iit/sources/robotology-superbuild/build/install/share/wbc/robots/iCubGazeboV2_5/yarprobotinterface.ini] (robot)
||| checking [/home/dferigo/.config/yarp/yarprobotinterface.ini] (YARP_CONFIG_HOME)
||| checking [/home/dferigo/.local/share/yarp/yarprobotinterface.ini] (YARP_DATA_HOME)
||| checking [/etc/yarp/yarprobotinterface.ini] (YARP_CONFIG_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/yarp/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/iCub/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/ICUBcontrib/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/robotology/icub-tests/suites/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/human-gazebo/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/HumanDynamicsEstimation/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/Wearables/yarprobotinterface.ini] (YARP_DATA_DIRS)
||| checking [/iit/sources/robotology-superbuild/build/install/share/wbc/yarprobotinterface.ini] (yarp.d)
yarp: cannot read from yarprobotinterface.ini
||| finding file [config]
||| checking [/home/dferigo/launch-fbe-analogsens.xml] (pwd)
||| checking [/iit/sources/robotology-superbuild/build/install/share/yarp/robots/iCubGazeboV2_5/launch-fbe-analogsens.xml] (robot)
||| found /iit/sources/robotology-superbuild/build/install/share/yarp/robots/iCubGazeboV2_5/launch-fbe-analogsens.xml
[DEBUG]Reading file /iit/sources/robotology-superbuild/build/install/share/yarp/robots/iCubGazeboV2_5/launch-fbe-analogsens.xml 
[DEBUG]yarprobotinterface: using xml parser for DTD v3.x 
[DEBUG]Reading file /iit/sources/robotology-superbuild/build/install/share/yarp/robots/iCubGazeboV2_5/launch-fbe-analogsens.xml 
[DEBUG]Preprocessor complete in:  0 s 
yarp: Port /baseestimation/yarprobotinterface active at tcp://192.168.8.102:10062/
[INFO]startup phase starting... 
||| finding paths [plugins]
yarp: Port /baseestimation/torso/rpc:o active at tcp://192.168.8.102:10063/
yarp: Port /baseestimation/torso/command:o active at tcp://192.168.8.102:10064/
yarp: Port /baseestimation/torso/stateExt:i active at tcp://192.168.8.102:10065/
yarp: Sending output from /baseestimation/torso/rpc:o to /icubSim/torso/rpc:i using tcp
yarp: Sending output from /baseestimation/torso/command:o to /icubSim/torso/command:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Receiving input from /icubSim/torso/stateExt:o to /baseestimation/torso/stateExt:i using udp
yarp: Port /baseestimation/left_arm/rpc:o active at tcp://192.168.8.102:10066/
yarp: Port /baseestimation/left_arm/command:o active at tcp://192.168.8.102:10067/
yarp: Port /baseestimation/left_arm/stateExt:i active at tcp://192.168.8.102:10068/
yarp: Sending output from /baseestimation/left_arm/rpc:o to /icubSim/left_arm/rpc:i using tcp
yarp: Sending output from /baseestimation/left_arm/command:o to /icubSim/left_arm/command:i using udp
yarp: Receiving input from /icubSim/left_arm/stateExt:o to /baseestimation/left_arm/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/right_arm/rpc:o active at tcp://192.168.8.102:10069/
yarp: Port /baseestimation/right_arm/command:o active at tcp://192.168.8.102:10070/
yarp: Port /baseestimation/right_arm/stateExt:i active at tcp://192.168.8.102:10071/
yarp: Sending output from /baseestimation/right_arm/rpc:o to /icubSim/right_arm/rpc:i using tcp
yarp: Sending output from /baseestimation/right_arm/command:o to /icubSim/right_arm/command:i using udp
yarp: Receiving input from /icubSim/right_arm/stateExt:o to /baseestimation/right_arm/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/left_leg/rpc:o active at tcp://192.168.8.102:10072/
yarp: Port /baseestimation/left_leg/command:o active at tcp://192.168.8.102:10073/
yarp: Port /baseestimation/left_leg/stateExt:i active at tcp://192.168.8.102:10074/
yarp: Sending output from /baseestimation/left_leg/rpc:o to /icubSim/left_leg/rpc:i using tcp
yarp: Sending output from /baseestimation/left_leg/command:o to /icubSim/left_leg/command:i using udp
yarp: Receiving input from /icubSim/left_leg/stateExt:o to /baseestimation/left_leg/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/right_leg/rpc:o active at tcp://192.168.8.102:10075/
yarp: Port /baseestimation/right_leg/command:o active at tcp://192.168.8.102:10076/
yarp: Port /baseestimation/right_leg/stateExt:i active at tcp://192.168.8.102:10077/
yarp: Sending output from /baseestimation/right_leg/rpc:o to /icubSim/right_leg/rpc:i using tcp
yarp: Sending output from /baseestimation/right_leg/command:o to /icubSim/right_leg/command:i using udp
yarp: Receiving input from /icubSim/right_leg/stateExt:o to /baseestimation/right_leg/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/head/rpc:o active at tcp://192.168.8.102:10078/
yarp: Port /baseestimation/head/command:o active at tcp://192.168.8.102:10079/
yarp: Port /baseestimation/head/stateExt:i active at tcp://192.168.8.102:10080/
yarp: Sending output from /baseestimation/head/rpc:o to /icubSim/head/rpc:i using tcp
yarp: Sending output from /baseestimation/head/command:o to /icubSim/head/command:i using udp
yarp: Receiving input from /icubSim/head/stateExt:o to /baseestimation/head/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
||| finding paths [plugins]
yarp: Port /baseestimation/icubSim/head/rpc:o active at tcp://192.168.8.102:10081/
yarp: Port /baseestimation/icubSim/head/command:o active at tcp://192.168.8.102:10082/
yarp: Port /baseestimation/icubSim/head/stateExt:i active at tcp://192.168.8.102:10083/
yarp: Sending output from /baseestimation/icubSim/head/rpc:o to /icubSim/head/rpc:i using tcp
yarp: Sending output from /baseestimation/icubSim/head/command:o to /icubSim/head/command:i using udp
yarp: Receiving input from /icubSim/head/stateExt:o to /baseestimation/icubSim/head/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/icubSim/torso/rpc:o active at tcp://192.168.8.102:10084/
yarp: Port /baseestimation/icubSim/torso/command:o active at tcp://192.168.8.102:10085/
yarp: Port /baseestimation/icubSim/torso/stateExt:i active at tcp://192.168.8.102:10086/
yarp: Sending output from /baseestimation/icubSim/torso/rpc:o to /icubSim/torso/rpc:i using tcp
yarp: Sending output from /baseestimation/icubSim/torso/command:o to /icubSim/torso/command:i using udp
yarp: Receiving input from /icubSim/torso/stateExt:o to /baseestimation/icubSim/torso/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/icubSim/left_arm/rpc:o active at tcp://192.168.8.102:10087/
yarp: Port /baseestimation/icubSim/left_arm/command:o active at tcp://192.168.8.102:10088/
yarp: Port /baseestimation/icubSim/left_arm/stateExt:i active at tcp://192.168.8.102:10089/
yarp: Sending output from /baseestimation/icubSim/left_arm/rpc:o to /icubSim/left_arm/rpc:i using tcp
yarp: Sending output from /baseestimation/icubSim/left_arm/command:o to /icubSim/left_arm/command:i using udp
yarp: Receiving input from /icubSim/left_arm/stateExt:o to /baseestimation/icubSim/left_arm/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/icubSim/right_arm/rpc:o active at tcp://192.168.8.102:10090/
yarp: Port /baseestimation/icubSim/right_arm/command:o active at tcp://192.168.8.102:10091/
yarp: Port /baseestimation/icubSim/right_arm/stateExt:i active at tcp://192.168.8.102:10092/
yarp: Sending output from /baseestimation/icubSim/right_arm/rpc:o to /icubSim/right_arm/rpc:i using tcp
yarp: Sending output from /baseestimation/icubSim/right_arm/command:o to /icubSim/right_arm/command:i using udp
yarp: Receiving input from /icubSim/right_arm/stateExt:o to /baseestimation/icubSim/right_arm/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/icubSim/left_leg/rpc:o active at tcp://192.168.8.102:10093/
yarp: Port /baseestimation/icubSim/left_leg/command:o active at tcp://192.168.8.102:10094/
yarp: Port /baseestimation/icubSim/left_leg/stateExt:i active at tcp://192.168.8.102:10095/
yarp: Sending output from /baseestimation/icubSim/left_leg/rpc:o to /icubSim/left_leg/rpc:i using tcp
yarp: Sending output from /baseestimation/icubSim/left_leg/command:o to /icubSim/left_leg/command:i using udp
yarp: Receiving input from /icubSim/left_leg/stateExt:o to /baseestimation/icubSim/left_leg/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
yarp: Port /baseestimation/icubSim/right_leg/rpc:o active at tcp://192.168.8.102:10096/
yarp: Port /baseestimation/icubSim/right_leg/command:o active at tcp://192.168.8.102:10097/
yarp: Port /baseestimation/icubSim/right_leg/stateExt:i active at tcp://192.168.8.102:10098/
yarp: Sending output from /baseestimation/icubSim/right_leg/rpc:o to /icubSim/right_leg/rpc:i using tcp
yarp: Sending output from /baseestimation/icubSim/right_leg/command:o to /icubSim/right_leg/command:i using udp
yarp: Receiving input from /icubSim/right_leg/stateExt:o to /baseestimation/icubSim/right_leg/stateExt:i using udp
[INFO]created device <remote_controlboard>. See C++ class RemoteControlBoard for documentation.
[INFO]created device <remotecontrolboardremapper>. See C++ class RemoteControlBoardRemapper for documentation.
||| finding paths [plugins]
yarp: Port /baseestimation/head/imu:i active at tcp://192.168.8.102:10099/
yarp: Receiving input from /icubSim/inertial to /baseestimation/head/imu:i using udp
[INFO]created device <genericSensorClient>. See C++ class yarp::dev::GenericSensorClient for documentation.
||| finding paths [plugins]
yarp: Port /baseestimation/l_arm_ft_sensor active at tcp://192.168.8.102:10100/
yarp: Port /baseestimation/l_arm_ft_sensor/rpc:o active at tcp://192.168.8.102:10101/
yarp: Receiving input from /icubSim/left_arm/analog:o to /baseestimation/l_arm_ft_sensor using udp
yarp: Sending output from /baseestimation/l_arm_ft_sensor/rpc:o to /icubSim/left_arm/analog:o/rpc:i using tcp
[INFO]created device <analogsensorclient>. See C++ class AnalogSensorClient for documentation.
yarp: Port /baseestimation/r_arm_ft_sensor active at tcp://192.168.8.102:10102/
yarp: Port /baseestimation/r_arm_ft_sensor/rpc:o active at tcp://192.168.8.102:10103/
yarp: Receiving input from /icubSim/right_arm/analog:o to /baseestimation/r_arm_ft_sensor using udp
yarp: Sending output from /baseestimation/r_arm_ft_sensor/rpc:o to /icubSim/right_arm/analog:o/rpc:i using tcp
[INFO]created device <analogsensorclient>. See C++ class AnalogSensorClient for documentation.
yarp: Port /baseestimation/l_leg_ft_sensor active at tcp://192.168.8.102:10104/
yarp: Port /baseestimation/l_leg_ft_sensor/rpc:o active at tcp://192.168.8.102:10105/
yarp: Receiving input from /icubSim/left_leg/analog:o to /baseestimation/l_leg_ft_sensor using udp
yarp: Sending output from /baseestimation/l_leg_ft_sensor/rpc:o to /icubSim/left_leg/analog:o/rpc:i using tcp
[INFO]created device <analogsensorclient>. See C++ class AnalogSensorClient for documentation.
yarp: Port /baseestimation/r_leg_ft_sensor active at tcp://192.168.8.102:10106/
yarp: Port /baseestimation/r_leg_ft_sensor/rpc:o active at tcp://192.168.8.102:10107/
yarp: Receiving input from /icubSim/right_leg/analog:o to /baseestimation/r_leg_ft_sensor using udp
yarp: Sending output from /baseestimation/r_leg_ft_sensor/rpc:o to /icubSim/right_leg/analog:o/rpc:i using tcp
[INFO]created device <analogsensorclient>. See C++ class AnalogSensorClient for documentation.
yarp: Port /baseestimation/l_foot_ft_sensor:i active at tcp://192.168.8.102:10108/
yarp: Port /baseestimation/l_foot_ft_sensor:i/rpc:o active at tcp://192.168.8.102:10109/
yarp: Receiving input from /icubSim/left_foot/analog:o to /baseestimation/l_foot_ft_sensor:i using udp
yarp: Sending output from /baseestimation/l_foot_ft_sensor:i/rpc:o to /icubSim/left_foot/analog:o/rpc:i using tcp
[INFO]created device <analogsensorclient>. See C++ class AnalogSensorClient for documentation.
yarp: Port /baseestimation/r_foot_ft_sensor:i active at tcp://192.168.8.102:10110/
yarp: Port /baseestimation/r_foot_ft_sensor:i/rpc:o active at tcp://192.168.8.102:10111/
yarp: Receiving input from /icubSim/right_foot/analog:o to /baseestimation/r_foot_ft_sensor:i using udp
yarp: Sending output from /baseestimation/r_foot_ft_sensor:i/rpc:o to /icubSim/right_foot/analog:o/rpc:i using tcp
[INFO]created device <analogsensorclient>. See C++ class AnalogSensorClient for documentation.
||| finding paths [plugins]
[INFO]FrameTransformServer: transforms_lifetime set to: 0.2 
[INFO]FrameTransformServer: Enabled ROS publisher 
[INFO]FrameTransformServer: Enabled ROS subscriber 
yarp: Port /transformServer/rpc active at tcp://192.168.8.102:10112/
yarp: Port /transformServer/transforms:o active at tcp://192.168.8.102:10113/
yarp: Port /tfNode active at tcp://192.168.8.102:10114/
yarp: Port /tf+@/tfNode active at tcp://192.168.8.102:10115/
yarp: Port /tf_static+@/tfNode active at tcp://192.168.8.102:10116/
yarp: Port /tf-@/tfNode active at tcp://192.168.8.102:10117/
yarp: Receiving input from /tf+@/tfNode to /tf-@/tfNode using tcp
yarp: Sending output from /tf+@/tfNode to /tf-@/tfNode using tcp
yarp: Port /tf_static-@/tfNode active at tcp://192.168.8.102:10118/
[INFO]Transform server started 
yarp: Receiving input from /tf_static+@/tfNode to /tf_static-@/tfNode using tcp
yarp: Sending output from /tf_static+@/tfNode to /tf_static-@/tfNode using tcp
[INFO]transformServer: no starting tf found 
[INFO]created device <transformServer>. See C++ class FrameTransformServer for documentation.
||| finding paths [plugins]
yarp: Port /base-estimator/left_foot_cartesian_wrench:i active at tcp://192.168.8.102:10119/
[ERROR]floatingBaseEstimatorV1:   could not connect to  /wholeBodyDynamics/left_foot/cartesianEndEffectorWrench:o 
[ERROR]floatingBaseEstimatorV1:  Could not connect to wholebodydynamics 
yarpdev: ***ERROR*** driver <baseEstimatorV1> was found but could not open
[WARNING]Cannot open device baseestimator 
[WARNING]Cannot open device baseestimator 
[WARNING]There was some problem opening one or more devices. Please check the log and your configuration 
[ERROR]One or more devices failed opening... see previous log messages for more info 
yarp: Removing input from /tf+@/tfNode to /tf-@/tfNode
yarp: Removing output from /tf+@/tfNode to /tf-@/tfNode
yarp: Removing input from /tf_static+@/tfNode to /tf_static-@/tfNode
yarp: Removing output from /tf_static+@/tfNode to /tf_static-@/tfNode
yarp: Removing output from /baseestimation/r_foot_ft_sensor:i/rpc:o to /icubSim/right_foot/analog:o/rpc:i
yarp: Removing input from /icubSim/right_foot/analog:o to /baseestimation/r_foot_ft_sensor:i
yarp: Removing output from /baseestimation/l_foot_ft_sensor:i/rpc:o to /icubSim/left_foot/analog:o/rpc:i
yarp: Removing input from /icubSim/left_foot/analog:o to /baseestimation/l_foot_ft_sensor:i
yarp: Removing output from /baseestimation/r_leg_ft_sensor/rpc:o to /icubSim/right_leg/analog:o/rpc:i
yarp: Removing input from /icubSim/right_leg/analog:o to /baseestimation/r_leg_ft_sensor
yarp: Removing output from /baseestimation/l_leg_ft_sensor/rpc:o to /icubSim/left_leg/analog:o/rpc:i
yarp: Removing input from /icubSim/left_leg/analog:o to /baseestimation/l_leg_ft_sensor
yarp: Removing output from /baseestimation/r_arm_ft_sensor/rpc:o to /icubSim/right_arm/analog:o/rpc:i
yarp: Removing input from /icubSim/right_arm/analog:o to /baseestimation/r_arm_ft_sensor
yarp: Removing output from /baseestimation/l_arm_ft_sensor/rpc:o to /icubSim/left_arm/analog:o/rpc:i
yarp: Removing input from /icubSim/left_arm/analog:o to /baseestimation/l_arm_ft_sensor
yarp: Removing input from /icubSim/inertial to /baseestimation/head/imu:i
yarp: Removing output from /baseestimation/icubSim/head/rpc:o to /icubSim/head/rpc:i
yarp: Removing output from /baseestimation/icubSim/head/command:o to /icubSim/head/command:i
yarp: Removing input from /icubSim/head/stateExt:o to /baseestimation/icubSim/head/stateExt:i
yarp: Removing output from /baseestimation/icubSim/torso/rpc:o to /icubSim/torso/rpc:i
yarp: Removing output from /baseestimation/icubSim/torso/command:o to /icubSim/torso/command:i
yarp: Removing input from /icubSim/torso/stateExt:o to /baseestimation/icubSim/torso/stateExt:i
yarp: Removing output from /baseestimation/icubSim/left_arm/rpc:o to /icubSim/left_arm/rpc:i
yarp: Removing output from /baseestimation/icubSim/left_arm/command:o to /icubSim/left_arm/command:i
yarp: Removing input from /icubSim/left_arm/stateExt:o to /baseestimation/icubSim/left_arm/stateExt:i
yarp: Removing output from /baseestimation/icubSim/right_arm/rpc:o to /icubSim/right_arm/rpc:i
yarp: Removing output from /baseestimation/icubSim/right_arm/command:o to /icubSim/right_arm/command:i
yarp: Removing input from /icubSim/right_arm/stateExt:o to /baseestimation/icubSim/right_arm/stateExt:i
yarp: Removing output from /baseestimation/icubSim/left_leg/rpc:o to /icubSim/left_leg/rpc:i
yarp: Removing output from /baseestimation/icubSim/left_leg/command:o to /icubSim/left_leg/command:i
yarp: Removing input from /icubSim/left_leg/stateExt:o to /baseestimation/icubSim/left_leg/stateExt:i
yarp: Removing output from /baseestimation/icubSim/right_leg/rpc:o to /icubSim/right_leg/rpc:i
yarp: Removing output from /baseestimation/icubSim/right_leg/command:o to /icubSim/right_leg/command:i
yarp: Removing input from /icubSim/right_leg/stateExt:o to /baseestimation/icubSim/right_leg/stateExt:i
yarp: Removing output from /baseestimation/head/rpc:o to /icubSim/head/rpc:i
yarp: Removing output from /baseestimation/head/command:o to /icubSim/head/command:i
yarp: Removing input from /icubSim/head/stateExt:o to /baseestimation/head/stateExt:i
yarp: Removing output from /baseestimation/right_leg/rpc:o to /icubSim/right_leg/rpc:i
yarp: Removing output from /baseestimation/right_leg/command:o to /icubSim/right_leg/command:i
yarp: Removing input from /icubSim/right_leg/stateExt:o to /baseestimation/right_leg/stateExt:i
yarp: Removing output from /baseestimation/left_leg/rpc:o to /icubSim/left_leg/rpc:i
yarp: Removing output from /baseestimation/left_leg/command:o to /icubSim/left_leg/command:i
yarp: Removing input from /icubSim/left_leg/stateExt:o to /baseestimation/left_leg/stateExt:i
yarp: Removing output from /baseestimation/right_arm/rpc:o to /icubSim/right_arm/rpc:i
yarp: Removing output from /baseestimation/right_arm/command:o to /icubSim/right_arm/command:i
yarp: Removing input from /icubSim/right_arm/stateExt:o to /baseestimation/right_arm/stateExt:i
yarp: Removing output from /baseestimation/left_arm/rpc:o to /icubSim/left_arm/rpc:i
yarp: Removing output from /baseestimation/left_arm/command:o to /icubSim/left_arm/command:i
yarp: Removing input from /icubSim/left_arm/stateExt:o to /baseestimation/left_arm/stateExt:i
yarp: Removing output from /baseestimation/torso/rpc:o to /icubSim/torso/rpc:i
yarp: Removing output from /baseestimation/torso/command:o to /icubSim/torso/command:i
yarp: Removing input from /icubSim/torso/stateExt:o to /baseestimation/torso/stateExt:i
[ERROR]Error in startup phase... see previous messages for more info 
[WARNING]Interrupt # 1 # received. 
[INFO]Interrupt received. Stopping all running threads. 
[INFO]interrupt1 phase starting... 
[INFO]interrupt1 phase finished. 
[INFO]shutdown phase starting... 
[INFO]Entering action level 2 of phase shutdown 
[ERROR]baseestimator is not a wrapper, therefore it cannot have detach actions 
[ERROR]Cannot run detach action on device baseestimator 
[INFO]All actions for action level 2 of shutdown phase started. Waiting for unfinished actions. 
[INFO]All actions for action level 2 of shutdown phase finished. 
[WARNING]There was some problem running actions for shutdown phase . Please check the log and your configuration 
[INFO]shutdown phase finished. 
[ERROR]Error in shutdown phase... see previous messages for more info 
[INFO]RFModule failed to open.
yarp: Destroying network clock
Success: port-to-port persistent connection added.
yarp: Removing input from /clock to /IITICUBLAP110/yarprobotinterface/31105/clock:i
```

</details>

<details>
<summary>Click to expand the error section</summary>

```
[ERROR]floatingBaseEstimatorV1:   could not connect to  /wholeBodyDynamics/left_foot/cartesianEndEffectorWrench:o 
[ERROR]floatingBaseEstimatorV1:  Could not connect to wholebodydynamics 
yarpdev: ***ERROR*** driver <baseEstimatorV1> was found but could not open
[WARNING]Cannot open device baseestimator 
[WARNING]Cannot open device baseestimator 
[WARNING]There was some problem opening one or more devices. Please check the log and your configuration 
[ERROR]One or more devices failed opening... see previous log messages for more info 
```

</details>

I'm trying to go through the code. It seems that the yarprobotinterface configuration is self-contained and already takes care to open all the depending devices, particularly:

- All the remote control boards and control board remapper (I guess used by wbd)
- The head IMU device (I guess used by the estimator)
- The F/T sensors (I guess used by wbd)

The error is raised by the following code:

https://github.com/robotology/whole-body-estimators/blob/38dc6a269f1b0bce09f31d8ae9c4ca3602e8e46b/devices/baseEstimatorV1/src/fbeRobotInterface.cpp#L432-L438

More details:

- Ubuntu 18.04
- Gazebo 11
- robotology-superbuild updated today

cc @prashanthr05 @traversaro

### PR DESCRIPTION
**Pull request by prashanthr05**
This PR updates the documentation for launching the `baseEstimatorv1`.

Fixes #76.

### CODE REVIEW

File: devices/baseEstimatorV1/README.md [hunk 1]
```diff
@@ -18,21 +18,26 @@ This estimator uses the kinematics, IMU mounted on the head and the contact forc
...
+     YARP_CLOCK=/clock yarprobotinterface --config launch-wholebodydynamics.xml
     ```
-    This launches both the floating base estimation device and the whole body dynamics device.
 5. Reset the offset of the FT sensors. Open a terminal and write
 
    ```
    yarp rpc /wholeBodyDynamics/rpc
    >> resetOffset all
    ```
+   Note that this step can be avoided if a flag `startWithZeroFTSensorOffsets` is set to true in the `wholeBodyDynamics` configuration file (This is applicable only in the simulation.).q
```
Comment (HosameldinMohamed) [inline devices/baseEstimatorV1/README.md:32]: typo at the end of the line?

Comment (prashanthr05) [reply devices/baseEstimatorV1/README.md]: Fixed in https://github.com/robotology/whole-body-estimators/commit/1a36d9fb2d9eb625d56755fc2a12fe5751340d72.

> Do you think updating the CHANGELOG.md file is necessary in this case?

I think it's not necessary. What would be your opinion on this?

Comment (HosameldinMohamed) [reply devices/baseEstimatorV1/README.md]: You're right. I just wanted to see what you think about it.

It makes sense not to put it.. It's just a change in the documentation.

**Comment by prashanthr05** [comment]
Merged, thanks! :)

**Pull request merged**